pkg/platform/runtime/target/target.go

```diff
@@ -50,40 +49,12 @@ const (
 	triggerUnknown            Trigger = "unknown"
 )
 
-// usageTriggers are triggers that indicate actual usage of the runtime (as oppose to simply making changes to the runtime)
-var usageTriggers = []Trigger{
-	TriggerActivate,
-	TriggerScript,
-	TriggerDeploy,
-	TriggerExec,
-	TriggerExecutor,
-	TriggerSwitch,
-	TriggerImport,
-	TriggerInit,
-	TriggerPackage,
-	TriggerPull,
-	TriggerReset,
-	TriggerRevert,
-	TriggerShell,
-	TriggerCheckout,
-	TriggerUse,
-	TriggerOfflineInstaller,
-	TriggerOfflineUninstaller,
-}
-
 func NewExecTrigger(cmd string) Trigger {
 	return Trigger(fmt.Sprintf("%s: %s", TriggerExec, cmd))
 }
 
 func (t Trigger) IndicatesUsage() bool {
-	if funk.Contains(usageTriggers, t) {
-		return true
-	}
-	return t.IsExecTrigger() && funk.Contains(usageTriggers, TriggerExec)
-}
-
-func (t Trigger) IsExecTrigger() bool {
-	return strings.HasPrefix(string(t), string(TriggerExec)+": ")
+	return !strings.EqualFold(string(t), string(TriggerRefresh))
```

Comment:
It is not clear that this is a blacklist. Also, I thought we wanted to consider `state refresh` as a runtime trigger (bullet 3 in the JIRA ticket). Furthermore, AC 2 wants to ensure we don't trigger use on state tool update. I'm not positive this is addressed here.

Reply:
Whoops, I used the wrong trigger. I think updating it should address the issues you've raised but please let me know if I've missed something.